src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/course/restapi/CourseController.kt

```diff
@@ -15,7 +15,7 @@ import java.util.UUID
 import org.springframework.beans.factory.annotation.Autowired
 
 @Service
-class CoursesController
+class CourseController
```

Comment:
Interesting - this is different to how we tend to call things on the frontend. Is this consistent with other MOJ projects or the Approved Premises API?

Reply:
I've found this to be somewhat inconsistent.

For instance, the Approved Premises API both has `AssessmentController` for an `AssessmentEntity` but also has `UsersController` for a `UserEntity`. However, the overwhelming majority of these work with the singular, not the plural, which is why I've opted for it here.

Reply:
Makes sense - happy to go with one over the other, as long as we stick to it consistently